### PR TITLE
feat: add missing pages and remove unused views

### PR DESCRIPTION
**Pull request by tribiec**
This PR adds missing views, remove unused views, methods and captions.
### Status
- Webpay Plus
- Full Transaction
- Full Transaction Mall
### Refund
- Full Transaction
- Full Transaction Mall
### Deleted (Inscription Deleted)
- Oneclick
### Commit
- Full Transaction Deferred

Also we fixed some wrong captions.

## Evidence
- https://github.com/TransbankDevelopers/transbank-sdk-python-webpay-rest-example/assets/16856300/796017ea-818d-4227-b0ed-6cf7ba1843ef
- https://github.com/TransbankDevelopers/transbank-sdk-python-webpay-rest-example/assets/16856300/4b7c3818-d7e3-4bbf-8a49-afe5931ee853

### CODE REVIEW

**Review by victormendoza96 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -9,10 +8,18 @@ You need to have `docker` and `docker-compose`installed
...
 `volumes` entry, to verify that the origin path (the one with `{$HOME}`) matches the location of the SDK on your local
-machine
\ No newline at end of file
+machine
+
+### Alternative Way to run the example project
+
+If you don't want to use docker, you can run the example project by following these steps:
+
+- Execute the command python3 -m venv venv
```
Comment (Matiasnickolas) [inline README.md]: this commands under the new section should be between backtips for a better readability

File: README.md [hunk 2]
```diff
@@ -9,10 +8,18 @@ You need to have `docker` and `docker-compose`installed
...
+- Activate the virtual environment with source venv/bin/activate
+- Install the dependencies with pip install transbank-sdk
```
Comment (Matiasnickolas) [inline README.md]: this installs only the transbank-sdk, but you need all from requirements.txt
replace this command with _pip install -r requirements.txt_

File: oneclick_deferred/routes.py [hunk 3]
```diff
@@ -206,4 +150,5 @@ def capture():
         "commerce_code": commerce_code,
         "capture_amount": capture_amount
     }
-    return render_template('oneclick/deferred/capture.html', request= request_data, response=response)
\ No newline at end of file
+    return render_template('oneclick/deferred/capture.html', request= request_data, response=response)
+
```
Comment (Matiasnickolas) [inline oneclick_deferred/routes.py]: missing empty line

File: templates/transaccion_completa/refund-form.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+                    <label>amount:</label><input name="amount" type="text" class="form-control"/>
+                </div>
+                <input name="enviar" type="submit" value="Enviar" class="btn btn-primary"/>
+        </form>
+    </div>
+    </div>
+<br>
+<a href="/">&laquo; volver a index</a>
+</body>
+</html>
```
Comment (Matiasnickolas) [inline templates/transaccion_completa/refund-form.html]: missing empty line

File: templates/transaccion_completa/status-form.html [hunk 5]
```diff
@@ -0,0 +1,37 @@
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <title>Ejemplos Webpay Plus - Solicitar Estado</title>
```
Comment (Matiasnickolas) [inline templates/transaccion_completa/status-form.html]: template name is for product `complete transaction` but this title is `Webpay plus`

File: templates/transaccion_completa/status-form.html [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    {% include 'include/header.html' %}
+  </head>
+
+  <body class="container">
+    <nav aria-label="breadcrumb">
+      <ol class="breadcrumb">
+        <li class="breadcrumb-item">
+          <a href="#">Ejemplos Webpay Plus - Solicitar Estado</a>
```
Comment (Matiasnickolas) [inline templates/transaccion_completa/status-form.html]: same here, template name is for product `complete transaction` but this title is `Webpay plus`

File: templates/transaccion_completa_mall/refund-form.html [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+                    <label>Amount:</label><input name="amount" class="form-control"/>
+                </div>
+                <input name="enviar" type="submit" value="Enviar" class="btn btn-primary"/>
+        </form>
+    </div>
+    </div>
+<br>
+<a href="/">&laquo; Volver a index</a>
+</body>
+</html>
```
Comment (Matiasnickolas) [inline templates/transaccion_completa_mall/refund-form.html]: missing empty line

File: templates/transaccion_completa_mall/status-form.html [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+                <label>token_ws:</label><input name="token_ws" type="text" class="form-control"/>
+            </div>&nbsp;&nbsp;&nbsp;
+            <input name="enviar" type="submit" value="Enviar" class="btn btn-primary"/>
+        </form>
+    </div>
+</div>
+<br>
+<a href="/">&laquo; volver a index</a>
+</body>
+</html>
```
Comment (Matiasnickolas) [inline templates/transaccion_completa_mall/status-form.html]: missing empty line

File: transaccion_completa_deferred/routes.py [hunk 9]
```diff
@@ -45,13 +45,7 @@ def installments():
...
     grace_period = 'false'
     commit = tx.commit(token, id_query_installments, deferred_period_index, grace_period)
-    request_data = {
-        "token": token,
-        "id_query_installments": id_query_installments,
-        "deferred_period_index": deferred_period_index,
-        "grace_period": grace_period
-    }
-    return render_template('transaccion_completa/deferred/commit.html', request= request_data, response=commit)
+    return render_template('transaccion_completa/deferred/commit.html', request=commit, response=commit, token=token)
```
Comment (Matiasnickolas) [inline transaccion_completa_deferred/routes.py]: why commit is assigned to _request_ and _response_?

File: transaccion_completa_deferred/routes.py [hunk 10]
```diff
@@ -139,4 +72,5 @@ def capture():
         "authorization_code": authorization_code,
         "capture_amount": capture_amount
     }
-    return render_template('transaccion_completa/deferred/capture.html', request= request_data, response=response)
\ No newline at end of file
+    return render_template('transaccion_completa/deferred/capture.html', request= request_data, response=response)
+
```
Comment (Matiasnickolas) [inline transaccion_completa_deferred/routes.py]: missing empty line

File: transaccion_completa_mall_deferred/routes.py [hunk 11]
```diff
@@ -204,4 +136,5 @@ def capture():
         "buy_order": buy_order,
         "commerce_code": commerce_code
     }
-    return render_template('/transaccion_completa_mall/deferred/capture.html', request=request_data, response=response)
\ No newline at end of file
+    return render_template('/transaccion_completa_mall/deferred/capture.html', request=request_data, response=response)
+
```
Comment (Matiasnickolas) [inline transaccion_completa_mall_deferred/routes.py]: missing empty line

File: webpay_plus/routes.py [hunk 12]
```diff
@@ -70,4 +70,18 @@ def webpay_plus_refund():
...
+    return render_template('webpay/plus/status-form.html')
+
+
+@bp.route('status', methods=['POST'])
+def status():
+    token_ws = request.form.get('token_ws')
+    tx = Transaction()
+    resp = tx.status(token_ws)
+    return render_template('webpay/plus/status.html', response=resp, token=token_ws, req=request.form)
+
```
Comment (Matiasnickolas) [inline webpay_plus/routes.py]: missing empty line

File: webpay_plus_mall/routes.py [hunk 13]
```diff
@@ -95,4 +95,5 @@ def patpass_webpay_status():
         response = (MallTransaction()).status(token)
         return render_template("webpay/plus_mall/status.html", token=token, response=response)
     except TransbankError as e:
-        print(e.message)
\ No newline at end of file
+        print(e.message)
+
```
Comment (Matiasnickolas) [inline webpay_plus_mall/routes.py]: missing empty line

File: templates/transaccion_completa_mall/status-form.html [hunk 14]
```diff
@@ -0,0 +1,28 @@
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <title>Ejemplos Transaccion Completa - Solicitar Estado</title>
```
Comment (Matiasnickolas) [inline templates/transaccion_completa_mall/status-form.html]: typo: `Transacción`

File: templates/transaccion_completa_mall/status-form.html [hunk 15]
```diff
@@ -0,0 +1,28 @@
...
+    {% include 'include/header.html' %}
+</head>
+
+<body class="container">
+<nav aria-label="breadcrumb">
+    <ol class="breadcrumb">
+        <li class="breadcrumb-item"><a href="#">Ejemplos Transaccion Completa - Solicitar Estado</a></li>
```
Comment (Matiasnickolas) [inline templates/transaccion_completa_mall/status-form.html]: typo: `Transacción`

File: templates/transaccion_completa_mall/refund-form.html [hunk 16]
```diff
@@ -0,0 +1,38 @@
...
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <title>Ejemplo Transaccion Completa - Transacci&oacute;n Anulaci&oacute;n</title>
+    {% include 'include/header.html' %}
+</head>
+
+<body class="container">
+    <nav aria-label="breadcrumb">
+    <ol class="breadcrumb">
+    <li class="breadcrumb-item"><a href="#">Ejemplo Transaccion Completa - Transaccion Anulaci&oacute;n</a></li>
```
Comment (Matiasnickolas) [inline templates/transaccion_completa_mall/refund-form.html]: typo: `Transacción`

File: templates/transaccion_completa_mall/refund-form.html [hunk 17]
```diff
@@ -0,0 +1,38 @@
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <title>Ejemplo Transaccion Completa - Transacci&oacute;n Anulaci&oacute;n</title>
```
Comment (Matiasnickolas) [inline templates/transaccion_completa_mall/refund-form.html]: typo: `Transacción`

File: templates/transaccion_completa/transaction_status.html [hunk 18]
```diff
@@ -1,7 +1,27 @@
...
-{{ req }}
-
-<h2>Response</h2>
-{{ resp }}
\ No newline at end of file
+  <body class="container">
+    <nav aria-label="breadcrumb">
+      <ol class="breadcrumb">
+        <li class="breadcrumb-item">
+          <a href="#">Ejemplos Transaccion Completa - Status de Transaccionn</a>
```
Comment (Matiasnickolas) [inline templates/transaccion_completa/transaction_status.html]: typo: `Transacción`

File: templates/transaccion_completa/transaction_refunded.html [hunk 19]
```diff
@@ -1,7 +1,35 @@
...
+    <div class="alert alert-warning" role="alert">
+      <h3>request</h3>
+      token_ws: {{ token }}, amount: {{ amount }}
+    </div>
+    <div class="alert alert-primary" role="alert">
+      <h3>result</h3>
+      {{ resp }}
+    </div>
+    <br />
+    <p><samp>Reembolso realizado con exito.</samp></p>
```
Comment (Matiasnickolas) [inline templates/transaccion_completa/transaction_refunded.html]: typo: `éxito`

File: templates/transaccion_completa/transaction_refunded.html [hunk 20]
```diff
@@ -1,7 +1,35 @@
...
+  </head>
 
-<h2>Request</h2>
-{{ req }}
+  <body class="container">
+    <nav aria-label="breadcrumb">
+      <ol class="breadcrumb">
+        <li class="breadcrumb-item">
+          <a href="#"
+            >Ejemplos Transaccion Completa - Reembolsar Transacci&oacute;nn</a
```
Comment (Matiasnickolas) [inline templates/transaccion_completa/transaction_refunded.html]: typo: `Transacción`

File: templates/transaccion_completa/refund-form.html [hunk 21]
```diff
@@ -0,0 +1,32 @@
...
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <title>Ejemplo Transaccion Completa - Transaccion Anulaci&oacute;n</title>
+    {% include 'include/header.html' %}
+</head>
+
+<body class="container">
+    <nav aria-label="breadcrumb">
+    <ol class="breadcrumb">
+    <li class="breadcrumb-item"><a href="#">Ejemplo Transaccion Completa - Transaccion Anulaci&oacute;n</a></li>
```
Comment (Matiasnickolas) [inline templates/transaccion_completa/refund-form.html]: typo: `Transacción`

File: templates/transaccion_completa/refund-form.html [hunk 22]
```diff
@@ -0,0 +1,32 @@
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <title>Ejemplo Transaccion Completa - Transaccion Anulaci&oacute;n</title>
```
Comment (Matiasnickolas) [inline templates/transaccion_completa/refund-form.html]: typo: `Transacción`

File: templates/transaccion_completa/deferred/commit.html [hunk 23]
```diff
@@ -1,14 +1,14 @@
...
-    <title>Ejemplo Transaccion completa Diferido- Commit</title>
+    <title>Ejemplo Transaccion Completa Diferido - Commit</title>
     {% include 'include/header.html' %}
 </head>
 
 <body class="container">
 <nav aria-label="breadcrumb">
     <ol class="breadcrumb">
-        <li class="breadcrumb-item"><a href="#">Ejemplo Transaccion completa Diferido- Commit</a></li>
+        <li class="breadcrumb-item"><a href="#">Ejemplo Transaccion Completa Diferido - Commit</a></li>
```
Comment (Matiasnickolas) [inline templates/transaccion_completa/deferred/commit.html]: typo: `Transacción`

File: templates/transaccion_completa/deferred/commit.html [hunk 24]
```diff
@@ -1,14 +1,14 @@
 <html>
 <head>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-    <title>Ejemplo Transaccion completa Diferido- Commit</title>
+    <title>Ejemplo Transaccion Completa Diferido - Commit</title>
```
Comment (Matiasnickolas) [inline templates/transaccion_completa/deferred/commit.html]: typo: `Transacción`

File: templates/oneclick/deleted.html [hunk 25]
```diff
@@ -1,26 +1,38 @@
...
+      {{ req }}
+    </div>
+    {% if resp == True %}
+    <div class="alert alert-primary" role="alert">
+    {% else %}
+    <div class="alert alert-danger" role="alert">
+    {% endif %}
+      <h3>result</h3>
+      {% if resp == True %}
+      <span>Inscripcion Eliminada </span>
```
Comment (Matiasnickolas) [inline templates/oneclick/deleted.html]: typo: `Inscripción`

File: templates/oneclick/deleted.html [hunk 26]
```diff
@@ -1,26 +1,38 @@
...
+      {% else %}
+      <span>Inscripcion No Eliminada</span>
```
Comment (Matiasnickolas) [inline templates/oneclick/deleted.html]: typo: `Inscripción`

File: templates/oneclick/deleted.html [hunk 27]
```diff
@@ -1,26 +1,38 @@
...
-<body class="container">
+  <body class="container">
     <nav aria-label="breadcrumb">
-    <ol class="breadcrumb">
-    <li class="breadcrumb-item"><a href="#">Ejemplo Oneclick Mall - Borrar Inscripcion</a></li>
-    <li class="breadcrumb-item active" aria-current="page">Step: Deleted Inscription</li>
-    </ol>
+      <ol class="breadcrumb">
+        <li class="breadcrumb-item">
+          <a href="#">Ejemplo Oneclick Mall - Borrar Inscripcion</a>
```
Comment (Matiasnickolas) [inline templates/oneclick/deleted.html]: typo: `Inscripción`

**Review by Matiasnickolas — APPROVED** [review]

**Pull request merged**